db/dcrpg/pgblockchain.go

```diff
@@ -4847,7 +4849,8 @@ func (pgb *ChainDB) GetAPITransaction(txid *chainhash.Hash) *apitypes.Tx {
 			Expiry:   txraw.Expiry,
 			Vin:      make([]apitypes.Vin, len(txraw.Vin)),
 			Vout:     make([]apitypes.Vout, len(txraw.Vout)),
-			Tree:     txhelpers.TxTree(msgTx, treasuryActive),
+			Tree:     txTree,
+			Type:     strings.ToLower(txhelpers.TxTypeToString(int(txTree))),
 		},
```

Comment:
I think it looks better in lowercase, no?

Reply:
Totally.  Just was commenting on that other thread.

Reply:
Thanks for the review.